stable/kanister-operator/README.md

```diff
@@ -0,0 +1,63 @@
+# kanister-operator
+
+[kanister-operator](https://github.com/kanisterio/kanister) is a Kubernetes operator for Kanister framework.
+
+Kanister is a framework that enables application-level data management on Kubernetes. It allows domain experts to capture application specific data management tasks via blueprints, which can be easily shared and extended. The framework takes care of the tedious details surrounding execution on Kubernetes and presents a homogeneous operational experience across applications at scale.
+
+## TL;DR;
+
+```console
+$ helm install stable/kanister-operator
+```
+
+## Introduction
+
+This chart bootstraps a kanister-operator deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.7+ with Beta APIs enabled
```

Comment:
1.8+

We officially support the current and previous minor versions of Kubernetes.